src/Access/UserPolicy.php

```diff
@@ -9,9 +18,9 @@ class UserPolicy extends AbstractPolicy
 {
     protected $model = User::class;
 
-    public function flagrowCanImpersonate(User $actor, User $user)
+    public function canImpersonate(User $actor, User $user)
```

Comment:
I always prefixed my permissions just in case another extension was to define a permission with the same name for a different purpose. Though in this case it's probably specific enough to be low risk.